makefiles/Makefile.devhub-content

```diff
@@ -62,13 +62,14 @@ next-gen-stage: ## Host online for review
 	@:
 
 next-gen-deploy:	
-	@yes | mut-publish public ${PRODUCTION_BUCKET} --prefix="$(filter-out $@,$(MAKECMDGOALS))" --deploy --deployed-url-prefix=https://docs.mongodb.com --json --all-subdirectories ${ARGS};
-	@echo "Hosted at ${PRODUCTION_URL}/$(filter-out $@,$(MAKECMDGOALS))/index.html";
-	$(MAKE) deploy-search-index
+	@yes | mut-publish public ${PRODUCTION_BUCKET} --prefix="$(filter-out $@,$(MAKECMDGOALS))" --deploy --deployed-url-prefix=https://developer.mongodb.com --json --all-subdirectories ${ARGS};
+	@echo "Hosted at ${PRODUCTION_URL}/index.html";
```

Comment:
Should deployed url prefix be the  PRODUCTION_URL var?

Reply:
we can do that - but that is not the case in the other makefiles that I used as a model

Reply:
what would you like to do in this case?

Reply:
even docs-landing? This should follow the docs-landing model.